### ISSUE TITLE
Go to implementation

### ISSUE THREAD

**Issue by TomSoderling** [mfractor/mfractor-feedback#1]
http://docs.mfractor.com/code-actions/csharp/#go-to-implementation

I'm not sure if I'm seeing this option in VS for Mac to use this feature. I'll attach a screenshot. All I see is Navigate > Implementing Members, but it doesn't do anything.
Do I need to configure it  somehow?

I work in a large solution, so being able to navigate right to the implementation of an interface that lives in the same PCL project would be a big help.

<img width="576" alt="screen shot 2018-03-14 at 11 43 38 am" src="https://user-images.githubusercontent.com/18268845/37416940-1ad866f4-277d-11e8-95f1-2cf0a4c02221.png">

Thanks!

**Comment by matthewrdev** [comment]
Hi Tom,

The **Implementing Members** is not an MFractor code action. The Go To Implementation shortcut is accessed through the **Option** + **D** keyboard binding.

**Comment by TomSoderling** [comment]
Oh, I see. So it's not in the context menu at all.
Looks like I had something bound to Option + D already. Now it's working! Very nice.

BTW, I noticed that the MFractor keybindings conflict with several standard VS key bindings - the most important being ⌘ + D, Go To Definition

**Comment by matthewrdev** [comment]
Yes, it's not ideal that we conflict with other key bindings like ⌘ + D and Option + Return. We are a bit stuck here as Visual Studio Mac doesn't have appropriate extensibility in place for us to modify the behaviour of these key-bindings without adding our own implementation on top of the existing key bindings.

In a perfect world, we wouldn't need to do this. Unfortunately, it's just the way that VS Mac is for the moment!

**Comment by matthewrdev** [comment]
@TomSoderling Is this issue fixed for you? If so, please feel free to close it :)
